install.md

```diff
@@ -195,6 +195,96 @@ echo 'deb [signed-by=/usr/share/keyrings/projectatomic-ppa.gpg] http://ppa.launc
 apt update
 apt -y install -t stretch-backports libostree-dev golang
 apt -y install bats btrfs-tools git libapparmor-dev libdevmapper-dev libglib2.0-dev libgpgme11-dev libseccomp-dev libselinux1-dev skopeo-containers go-md2man
-``` 
+```
```

Comment:
hmm, surprised this didn't get dinged by git validation.

Reply:
I don't see this in my code now.

Reply:
Yeah,  the space at the end of the three ticks is gone now, just surprised it got through without getting flagged in prior PR's.